conceptnet5/builders/glove.py

```diff
@@ -1,16 +1,85 @@
 import sys
 
+from ftfy import fix_text
+
 import numpy as np
 
-from retrofit_glove import load_glove_vectors
+from sklearn.preprocessing import normalize
 from assoc_space import AssocSpace, LabelSet
-from assoc_space.eigenmath import normalize_rows
+from conceptnet5.nodes import normalized_concept_uri
+from wordfreq import word_frequency
+
+def conceptnet_normalizer(text):
+    """
+    Normalizes a text into a concept URI. This function assume the text is
+    english.
+    """
+    return normalized_concept_uri('en', text)
+
+def load_glove_vectors(filename, labels, filter_beyond_row=250000,
+                        end_row=1000000, frequency_cutoff=1e-6,
+                        verbose=10000, normalize=True):
```

Comment:
It seems better to fix this now than later: can we call the operation on word texts something besides 'normalize'? `standardize` would at least be different, or if we wanted to borrow a term from the Web-dev world, `slugify` is weird but describes exactly the operation.